test/common/upstream/outlier_detection_impl_test.cc

```diff
@@ -30,27 +31,39 @@ namespace Upstream {
 namespace Outlier {
 
 TEST(OutlierDetectorImplFactoryTest, NoDetector) {
-  Json::ObjectSharedPtr loader = Json::Factory::loadFromString("{}");
+  std::string json = R"EOF(
```

Comment:
Unfortunately, this doesn't work for JSON config fragments that still need to be passed through the cluster manager, so I'll just add some boilerplate string define instead of proto there.